src/table/table.tsx

```diff
@@ -211,10 +218,10 @@ class Table<ROW> extends React.Component<Props<ROW>, State> {
         return !action.tooltip ? (
             button
         ) : (
-            <Tooltip key={action.id} text={action.tooltip} placement="top">
-                {button}
-            </Tooltip>
-        );
+                <Tooltip key={action.id} text={action.tooltip} placement="top">
```

Comment:
Why has there been some formatting? @bedakb Do we have some tooling here?

Reply:
Ups no, but I've created PR that introduces Prettier https://github.com/Webtrekk/Ampel-UI/pull/17